pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+**Checklist for the submission of new PEtab problems**
+- [ ] The problem ID is in the format `{LAST_NAME_OF_FIRST_AUTHOR}_{ABBREVIATED_JOURNAL_NAME}{YEAR_OF_PUBLICATION}`
+- [ ] There is a GitHub issue for this problem
```

Comment:
Do we really need an issue? It makes sense if one starts to convert some published problem to PEtab, to avoid other people working on the same thing. But if somebody wants to submit their just published files, I don't think we need an issue for that.